main.js

```diff
@@ -0,0 +1,79 @@
+    images: './images/laptop_2.png',
+    languages: ['HTML', 'CSS', 'JavaScript'],
+    source: 'https://github.com/joseheco',
+  },
+  {
+    name: 'Project name goes here',
+    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisiUt aliquip ex ea commodo consequat.  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.',
+    images: './images/laptop_3.png',
+    languages: ['HTML', 'CSS', 'JavaScript'],
+    source: 'https://github.com/joseheco',
+  },
+  {
+    name: 'Project name goes here',
+    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisiUt aliquip ex ea commodo consequat.  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.',
+    images: './images/laptop_4.png',
+    languages: ['HTML', 'CSS', 'JavaScript'],
+    source: 'https://github.com/joseheco',
+  },
+  {
+    name: 'Project name goes here',
+    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisiUt aliquip ex ea commodo consequat.  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.',
+    images: './images/laptop_5.png',
+    languages: ['HTML', 'CSS', 'JavaScript'],
+    source: 'https://github.com/josehco',
+  },
+  {
+    name: 'Project name goes here',
+    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisiUt aliquip ex ea commodo consequat.  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.',
+    images: './images/laptop_6.png',
+    languages: ['HTML', 'CSS', 'JavaScript'],
+    source: 'https://github.com/joseheco',
+  },
+];
+
+let num = projects.length;
```

Comment:
Kindly display the popup windows in your next ``projects``. It's only working for your first one project.
![Screen Shot 2021-11-13 at 12 13 57](https://user-images.githubusercontent.com/22258274/141654677-488c1644-5f4a-4118-9f72-92bcb947974e.png)